server/game/cards/02-AoA/QuantumFingertrap.js

```diff
@@ -6,13 +6,14 @@ class QuantumFingertrap extends Card {
             targets: {
                 first: {
                     cardType: 'creature',
-                    controller: 'any'
+                    controller: 'any',
+                    cardCondition: (card) => !(card.controller.creaturesInPlay.length < 2 && (card.controller.opponent && card.controller.opponent.creaturesInPlay.length < 2))
```

Comment:
this confuses me

Reply:
It is unnecessary to select a card to swap if either player has 1 card each. This avoids unnecessary prompt.
It cannot be a condition in action because we can still use the artifact.

Reply:
Can you rewrite it in a way that makes that clearer?  having a ! at the start of an expression is usually pretty confusing

Reply:
Done